### ISSUE TITLE
IntelliJ config always enforces multi-line comments/blocks

### ISSUE THREAD

**Issue by dflemstr** [spotify/spotify-checkstyle-config#14]
Given this code:

```java
/** Does nothing. */
public void doNothing() {}
```

...the IntelliJ config re-formats it into this code:

```java
/**
 * Does nothing.
 */
public void doNothing() {
}
```

The first style is fine according to the Google Style Guide and shouldn't change.

Opening this as an Issue as I'm not familiar with the IntelliJ config syntax so I don't know the idiomatic way to fix it, and I want to give people a chance to bikeshed about this ;)

**Comment by udoprog** [comment]
So bikeshedding here, reading the style guide the second form (as you present it) seems to be slightly preferred, as the first one is referred to as "may be substituted" and comes second in their listing.

> 7.1.1. [..] The basic form is always acceptable. The single-line form may be substituted when there are no at-clauses present, and the entirety of the Javadoc block (including comment markers) can fit on a single line.

As for breaking up the `{}`, both are referred to as "equally acceptable" in `4.1.3 Empty blocks: may be concise`, so I don't see an issue with our formatter changing it. We are permitted to have a preference if we feel like it.

**Comment by dflemstr** [comment]
@udoprog I for one prefer the single-line form, so maybe we could make it leave things as-is?

**Comment by davidxia** [comment]
I also prefer the single-line form.

These are the two lines to add to the IntelliJ XML.

```
<option name="JD_DO_NOT_WRAP_ONE_LINE_COMMENTS" value="true" />
<option name="KEEP_SIMPLE_METHODS_IN_ONE_LINE" value="true" />
```

**Comment by mattnworb** [comment]
+1 to allowing single line Javadoc comments when they contain no at-clauses

**Comment by caipre** [comment]
This would be resolved by https://github.com/spotify/spotify-checkstyle-config/pull/19.

**Comment by caipre** [comment]
Merged #19, this is resolved.
